server/src/server.py

```diff
@@ -120,6 +121,33 @@ async def check_print_time(time_seconds: float):
         return "1"
     return "0"
 
+@access_server_router.post("/postBluetoothDevice", response_class=PlainTextResponse)
```

Comment:
Same comment as above (yh some of the old endpoints are badly named :grimacing: )